src/Core/JVM/Stream/ReflectionClassReader.php

```diff
@@ -1,18 +1,54 @@
 <?php
 namespace PHPJava\Core\JVM\Stream;
 
+use PHPJava\Core\JVM\Parameters\Runtime;
+
 class ReflectionClassReader implements StreamReaderInterface
 {
     /**
-     * @var resource
+     * @var \ReflectionClass
      */
     private $handle;
 
     /**
-     * @param resource $handle
+     * @param \ReflectionClass $handle
      */
     public function __construct($handle)
     {
         $this->handle = $handle;
     }
+
+    public function getFields()
```

Comment:
/**
 * @return \ReflectionField[][]
 */
public function getFields(): array

Reply:
Fields haves one array only.

Reply:
Yeah, I lazily copied and pasted from the methods one...

```php
/**
 * @return \ReflectionProperty[]
 */
public function getFields(): array
```

Reply:
I'll change it!